<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 255), stop:0.09 rgba(246, 189, 237, 255), stop:0.14 rgba(194, 207, 246, 255), stop:0.19 rgba(184, 160, 168, 255), stop:0.25 rgba(171, 186, 248, 255), stop:0.32 rgba(243, 248, 224, 255), stop:0.385 rgba(249, 162, 183, 255), stop:0.47 rgba(100, 115, 124, 255), stop:0.58 rgba(251, 205, 202, 255), stop:0.65 rgba(170, 128, 185, 255), stop:0.75 rgba(252, 222, 204, 255), stop:0.805 rgba(206, 122, 218, 255), stop:0.86 rgba(254, 223, 175, 255), stop:0.91 rgba(254, 236, 244, 255), stop:1 rgba(255, 191, 221, 255));</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(172, 53, 164);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_MENU">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Gill Sans Ultra Bold</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="accessibleName">
             <string notr="true"/>
            </property>
            <property name="accessibleDescription">
             <string notr="true"/>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Gill Sans Ultra Bold&quot;;
color:rgb(255, 255, 255);
bord: rgb(1, 227, 252);</string>
            </property>
            <property name="text">
             <string>MENU CNC</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>551</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bt_minimizar">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 170, 255)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imagenes/bajar.svg</normaloff>imagenes/bajar.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_restaurar">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 170, 255)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imagenes/maximizar.svg</normaloff>imagenes/maximizar.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_maximizar">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 170, 255)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imagenes/minimizar.svg</normaloff>imagenes/minimizar.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_cerrar">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 170, 255)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imagenes/cerrar.svg</normaloff>imagenes/cerrar.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="menu_lateral">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(246, 158, 252);

</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="bt_uno">
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Bauhaus 93&quot;;
color: rgb(255, 255, 255);
background-color: rgb(220, 178, 73);
alternate-background-color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string>ENCENDER       </string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>imagenes/eevee_icon-icons.com_67563.svg</normaloff>imagenes/eevee_icon-icons.com_67563.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_dos">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Bauhaus 93&quot;;
COLOR:rgb(255, 255, 255);
background-color: rgb(162, 252, 93);</string>
               </property>
               <property name="text">
                <string>SUBIR ARCIVO</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>imagenes/bulbasaur_icon-icons.com_67580.svg</normaloff>imagenes/bulbasaur_icon-icons.com_67580.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_tres">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Bauhaus 93&quot;;
COLOR:rgb(255, 255, 255);
background-color: rgb(87, 180, 252);</string>
               </property>
               <property name="text">
                <string>COMENZAR</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>imagenes/snorlax_icon-icons.com_67505.svg</normaloff>imagenes/snorlax_icon-icons.com_67505.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_cuatro">
               <property name="styleSheet">
                <string notr="true">font: 16pt &quot;Bauhaus 93&quot;;
color: rgb(255, 255, 255);
background-color: rgb(248, 110, 64);</string>
               </property>
               <property name="text">
                <string>CANCELAR</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>imagenes/charmander_icon-icons.com_67576.svg</normaloff>imagenes/charmander_icon-icons.com_67576.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_paginas">
            <property name="styleSheet">
             <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 255), stop:0.09 rgba(246, 189, 237, 255), stop:0.14 rgba(194, 207, 246, 255), stop:0.19 rgba(184, 160, 168, 255), stop:0.25 rgba(171, 186, 248, 255), stop:0.32 rgba(243, 248, 224, 255), stop:0.385 rgba(249, 162, 183, 255), stop:0.47 rgba(100, 115, 124, 255), stop:0.58 rgba(251, 205, 202, 255), stop:0.65 rgba(170, 128, 185, 255), stop:0.75 rgba(252, 222, 204, 255), stop:0.805 rgba(206, 122, 218, 255), stop:0.86 rgba(254, 223, 175, 255), stop:0.91 rgba(254, 236, 244, 255), stop:1 rgba(255, 191, 221, 255));


</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgb(246, 158, 252);
</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="bt_menu_uno">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(97, 252, 242);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>imagenes/left-arrows_icon-icons.com_70075.svg</normaloff>imagenes/left-arrows_icon-icons.com_70075.svg</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="bt_menu_dos">
                  <property name="styleSheet">
                   <string notr="true">
background-color: rgb(97, 252, 242);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>imagenes/right-arrows_icon-icons.com_69943.svg</normaloff>imagenes/right-arrows_icon-icons.com_69943.svg</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_11">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(169, 235, 255);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>imagenes/loupe_78956.svg</normaloff>imagenes/loupe_78956.svg</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_5">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 202, 250);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QToolBox" name="toolBox">
                  <property name="styleSheet">
                   <string notr="true">
QToolBox::tab{
	background-color: rgb(255, 0, 255);
font: 75 16pt &quot;Bauhaus 93&quot;;
color: rgb(255, 255, 255);
}

QToolBox::tab:selected{
	background-color: rgb(255, 0, 255);
font: 75 16pt &quot;Bauhaus 93&quot;;
color: rgb(0, 0, 0);
}</string>
                  </property>
                  <property name="currentIndex">
                   <number>2</number>
                  </property>
                  <property name="tabSpacing">
                   <number>7</number>
                  </property>
                  <widget class="QWidget" name="page_uno">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>589</width>
                     <height>457</height>
                    </rect>
                   </property>
                   <attribute name="icon">
                    <iconset>
                     <normaloff>imagenes/squirtle_icon-icons.com_67504.svg</normaloff>imagenes/squirtle_icon-icons.com_67504.svg</iconset>
                   </attribute>
                   <attribute name="label">
                    <string>Archivos</string>
                   </attribute>
                   <layout class="QVBoxLayout" name="verticalLayout"/>
                  </widget>
                  <widget class="QWidget" name="page_dos">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>589</width>
                     <height>457</height>
                    </rect>
                   </property>
                   <attribute name="icon">
                    <iconset>
                     <normaloff>imagenes/psyduck_icon-icons.com_67509.svg</normaloff>imagenes/psyduck_icon-icons.com_67509.svg</iconset>
                   </attribute>
                   <attribute name="label">
                    <string>Cargando</string>
                   </attribute>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <widget class="QWidget" name="widget" native="true">
                      <widget class="QLabel" name="label_uno">
                       <property name="geometry">
                        <rect>
                         <x>50</x>
                         <y>50</y>
                         <width>431</width>
                         <height>331</height>
                        </rect>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="page_tres">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>589</width>
                     <height>457</height>
                    </rect>
                   </property>
                   <attribute name="icon">
                    <iconset>
                     <normaloff>imagenes/pikachu_icon-icons.com_67535.svg</normaloff>imagenes/pikachu_icon-icons.com_67535.svg</iconset>
                   </attribute>
                   <attribute name="label">
                    <string>Cancelando</string>
                   </attribute>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QWidget" name="widget_2" native="true">
                      <widget class="QLabel" name="label_dos">
                       <property name="geometry">
                        <rect>
                         <x>110</x>
                         <y>40</y>
                         <width>321</width>
                         <height>281</height>
                        </rect>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
